lib/modules/dosomething/dosomething_campaign/dosomething_campaign.strongarm.inc

```diff
@@ -16,7 +16,7 @@ function dosomething_campaign_strongarm() {
   $strongarm->name = 'entity_translation_settings_node__campaign';
   $strongarm->value = array(
     'default_language' => 'xx-et-author',
-    'hide_language_selector' => 1,
+    'hide_language_selector' => 0,
```

Comment:
it's the language selector at the top of the nodes
![screen shot 2015-09-10 at 1 00 34 pm](https://cloud.githubusercontent.com/assets/645205/9794941/f742b3cc-57bb-11e5-97ca-3fb04fe44d9d.png)